api/docs/release.dox

```diff
@@ -242,6 +242,10 @@ Further non-compatibility-affecting changes include:
    functionality. (Issue #3129). #module_mapper_t replaces the following
    #raw2trace_t APIs: handle_custom_data(), do_module_parsing(),
    do_module_parsing_and_mapping(), and find_mapped_trace_address().
+ - Added trace_metadata_writer_t, a set of utility functions for writing trace
```

Comment:
Make it clear that this is part of drcachesim's raw2trace_t so we know what part of DR this involves: otherwise it is confusing as "trace" means something else (a superblock) for the rest of DR.

Reply:
done